libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/navigation/MapboxNavigator.java

```diff
@@ -14,6 +14,10 @@
 
   private final Navigator navigator;
 
+  MapboxNavigator() {
+    this(new Navigator());
```

Comment:
@danesfeder remove what constructor?

Reply:
@devotaaabel the `MapboxNavigator(Navigator)` constructor - is that still being used?

Reply:
@danesfeder It's being used by the other constructor. This is basically fake dependency injection so that we can use that test with mocks and it will touch the exact same code in both cases. A lot of apps/libraries use overloaded constructors.